src/utils/lnurl.ts

```diff
@@ -68,11 +68,21 @@ export const handleLnurlPay = async ({
 
 	const milliSats = Math.floor(amountSats * 1000);
 
-	const callbackRes = await lnurlPay({
-		params,
-		milliSats,
-		comment: 'Bitkit LNURL-Pay',
-	});
+	let callbackRes: any;
```

Comment:
To avoid `let channelStatusRes: any;`

```suggestion
try {
		const callbackRes = await lnurlPay({
			params,
			milliSats,
			comment: 'Bitkit LNURL-Pay',
		});

		if (callbackRes.isErr()) {
			showToast({
				type: 'error',
				title: i18n.t('other:lnurl_pay_error'),
				description: callbackRes.error.message,
			});
			return err(callbackRes.error.message);
		}

		return ok(callbackRes.value.pr);
	} catch (e) {
		showToast({
			type: 'error',
			title: i18n.t('other:lnurl_pay_error'),
			description: e.message,
		});
		return err(e.message);
	}
```